### DISCUSSION TITLE
Welcome to ui Discussions!

### DISCUSSION THREAD

**Discussion by jamespaulmuir** [ScriptHero/ui#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by georgialoper** [comment]
Compiling some resources related to this project:

- provider's cmm-dsl [repo](https://git.innova-partners.com/cmm-app/cmm-dsl), [storybook](https://pages.git.innova-partners.com/cmm-app/cmm-dsl/?path=/story/*), and fairly extensive [wiki](https://git.innova-partners.com/cmm-app/cmm-dsl/wiki)
- marketing's [storybook](https://pages.git.innova-partners.com/cmm/marketing-component-library/?path=/story/welcome--to-storybook) and [repo](https://git.innova-partners.com/cmm/marketing-component-library)
-  keeping an eye out for the recording of the Component library talk at VueConf [here](https://www.vuemastery.com/conferences/vueconf-us-2021/) and on [twitter](https://twitter.com/vueconfus), but here's the component library [lightning talk repo](https://github.com/areindl/vueconf-build-a-component-library)

**Comment by jamespaulmuir** [comment]
I learned a ton about Vue3 from this (translated from Chinese) site: https://translate.google.com/translate?sl=auto&tl=en&u=https%3A%2F%2Fvue3.chengpeiquan.com%2F
